app/models/apple/podcast_container.rb

```diff
@@ -54,22 +54,25 @@ def self.upsert_podcast_container(episode, row)
       external_id = row.dig('api_response', 'val', 'data', 'id')
 
       pc =
-        if pc = where(episode_id: episode.feeder_id,
-                      apple_episode_id: episode.apple_id,
-                      vendor_id: episode.audio_asset_vendor_id,
-                      external_id: external_id).first
+        if pc = where(apple_episode_id: episode.apple_id,
+                      external_id: external_id,
+                      episode_id: episode.feeder_id,
+                      vendor_id: episode.audio_asset_vendor_id,).first
```

Comment:
Okay, so just sorting here.  Plus a trailing comma (which i'm a bit surprised Rubocop allows, right before the `)`).